security/keycloak-oidc-client-reactive-basic/src/test/resources/test-realm-realm.json

```diff
@@ -57,6 +58,28 @@
         "test-user-role",
         "uma_protection"
       ]
+    },
```

Comment:
This was added to reactive but not to ordinary one. Why?

Reply:
In reactive I setting the `quarkus.oidc-client.grant-options.password.username` to the default user which is defined here for Keycloak.

For non reactive I am not setting it and using a serving account.

From my perspective I'm checking if this works with `grant` property set and not set.

If not set the realm service account should be used when it is enabled on the Keycloak side.